samcli/hook_packages/terraform/copy_terraform_built_artifacts.py

```diff
@@ -268,3 +264,61 @@ def cli_exit():
     except OSError as ex:
         LOG.error("Copy/Unzip unsuccessful!", exc_info=ex)
         cli_exit()
+
+
+def create_backend_override():
+    """
+    Copies and rename the override tf file from the metadata directory to the root
+    directory of the TF application.
+    """
+    override_src_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), TF_BACKEND_OVERRIDE_FILENAME)
+    override_dest_path = os.path.join(os.getcwd(), TF_BACKEND_OVERRIDE_FILENAME + ".tf")
+    shutil.copyfile(override_src_path, override_dest_path)
```

Comment:
Is it okay if the file is already there for subsequent runs? Should we include some error handling here?

Reply:
as stated above, I'd change to use `shutils.copy2`. The behaviour is still the same though - If dst specifies a file that already exists, it will be replaced